set_version.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
@kthoms what is the diff?

Reply:
The script now derives the versions from pom.xml. It asks to enter the new version and computes a proposal to use a minor version increment.

So you don't have to change the versions in the script.